### DISCUSSION TITLE
Feedback Emails separated by a comma

### DISCUSSION THREAD

**Discussion by dcfnet** [code4recovery/12-step-meeting-list#1215] [Q&A] [answered]
User Feedback Emails are configured with two recipients.
![image](https://github.com/code4recovery/12-step-meeting-list/assets/97912656/2c47aa65-d429-445b-bf71-60d86ddcc1f8)

When clicking on Update Meeting Info
![image](https://github.com/code4recovery/12-step-meeting-list/assets/97912656/f2818ad2-a634-4dce-9b11-81fc72326579)

I get an email email with the recipient addresses separated by a comma
altwebcoord@aavirginia.org,webcoord@aavirginia.org
In Outlook, this gives me a problem. I know to replace it with a semi-colon.  But, I'm worried about end users.

If this is a WP issue, I apologise. Also, I think I remember the Update Meeting Info button used to generate form that the user could fill out.

Please comment at your convenience.  Thanks!

David

**Comment by joshreisner** [comment, marked as answer]
hi @dcfnet yes the TSML UI appearance only has the capability of forming a `mailto` link because it is frontend-only. some sites i know have set up a feedback form on a page with gravity forms (or similar) and then pointed the update meeting info button to that address using these instructions: https://wordpress.org/plugins/12-step-meeting-list/#can%20i%20set%20a%20custom%20%22update%20meeting%20info%22%20button%20in%20tsml%20ui%3F

using commas appears to still be the [recommended way to specify multiple recipients](https://blog.hubspot.com/website/html-mailto) in a `mailto` link. i don't have access to outlook unfortunately to confirm that there's an issue there, but if you're concerned about your users you could also create a single email alias like `feedback@aavirginia.org` that forwards to both recipients

**Reply by dcfnet** [reply]
Thanks Josh as always! I will go with the email alias.
